src/rust/cxx-integration-test/cxx-rust-integration-test.c++

```diff
@@ -0,0 +1,261 @@
+#include <rust/cxx-integration-test/lib.rs.h>
+#include <rust/cxx-integration/lib.rs.h>
+#include <rust/cxx.h>
+
+#include <signal.h>
+#include <sys/wait.h>
+#include <unistd.h>
+
+#include <kj/async.h>
+#include <kj/test.h>
+
+#include <exception>
```

Comment:
Nit: I know this code is just ported from downstream but these two headers are unused right?